app/services/ror_service.rb

```diff
@@ -15,7 +15,7 @@ def call
     results = organizations
     return [] if results['number_of_results'].to_i.zero?
 
-    results['items'].map { |org| org.slice('id', 'name') }
+    results['items'].map { |org| RorOrg.new(org.slice('id', 'name', 'aliases', 'addresses', 'country', 'types')) }
```

Comment:
we need some extra fields from RoR API now, and our service now returns an array of model objects instead of an array of hashes straight from the API